components/pages/home/DailyQuote.js

```diff
@@ -1,30 +1,73 @@
+import React, { useState, useEffect, useContext } from "react";
 import styled from "styled-components";
 
+import { DataContext } from "@/contexts/dataContext.js";
+
 const StyledContainer = styled.div`
   display: block;
   justify-content: center;
   align-items: center;
   text-align: center;
   width: auto;
-  height: 30vh;
   margin-top: 175px;
-  padding: 20px;
+  padding: 50px 20px;
   background-color: #03314b;
   border-top: #032330 solid 2px;
   border-bottom: #032330 solid 2px;
   font-size: 17px;
 `;
 
 export default function DailyQuote() {
+  const { contentData } = useContext(DataContext);
+
+  const allQuotesArray = contentData;
```

Comment:
I don't understand why you do this, you could work with `contentData` as it is, couldn't you?

If you need the variable name `allQuotesArray`, you could rename the `contentData` while destructuring it in line 21.

Reply:
thanks thats a copy paste mistake